Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -629,6 +635,13 @@ namespace AzToolsFramework
                 Path fromPath;
                 if (isFolder)
                 {
+                    // There is currently a bug in AssetProcessorBatch that doesn't handle empty folders
+                    // This code is needed until that bug is fixed. GHI 13340
```

Comment:
uber nit: `GHI-13340` (this is the convention I've seen most often and will make grepping the code base easier in future)